web/src/components/organisms/Project/settings/hooks.ts

```diff
@@ -95,10 +95,11 @@ export default () => {
     async (alias: string) => {
       if (!alias) return false;
 
-      const response = await CheckProjectAlias({ variables: { alias } });
+      if (!workspaceId) throw new Error();
```

Comment:
Please add a descriptive error message for better debugging.

```suggestion
      if (!workspaceId) {
        throw new Error("Workspace ID is required to check project alias");
      }
```